### ISSUE TITLE
RSS feed on vanilla-java.github.io broken (404)

### ISSUE THREAD

**Issue by kno10** [peter-lawrey/Vanilla-Java#2]
`<link rel="alternate" type="application/rss+xml" title="Vanilla Java" href="https://vanilla-java.github.io/rss/" />`
Points to a 404 URL. But I'd prefer a full feed, instead of the "abstract only" version on your old blog. Thanks.

Comments also do not work ("unable to load Disqus").

**Comment by peter-lawrey** [comment]
I agree. This is a known issue with hub press.  I will ask them if they
have an update.
